src/redux/voting/actions.js

```diff
@@ -23,7 +37,11 @@ export const watchPoll = (notice: PollNoticeModel) => async (dispatch) => {
       dispatch(handlePollRemoved(notice.pollId()))
       break
     case IS_ACTIVATED:
+      dispatch(handlePollUpdated(notice.poll()), state.activePollsCount().plus(1))
+      break
     case IS_ENDED:
+      dispatch(handlePollUpdated(notice.poll()), state.activePollsCount().minus(1))
+      break
```

Comment:
use `return`

Reply:
no, because after switch we have one more dipatch